static/main.html

```diff
@@ -3,9 +3,13 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link rel="stylesheet" href="main.css" charset="utf-8">
+    <title>WebTorrent Desktop</title>
+    <link rel="stylesheet" href="main.css">
+    <script>
+	require('../build/renderer/main.js')
```

Comment:
Why is this no longer a script tag with an `async` property? I did LOTS of perf testing and found that that was fastest. Is this change required to get React to work?